### PR TITLE
Change steps for brief responses with non-summary tables

### PR DESCRIPTION
**Pull request by domoscargin**
https://trello.com/c/jgoHhWX3/284-2-replace-summary-table-in-opportunities-dashboard-in-brief-responses-frontend

### CODE REVIEW

File: features/supplier/supplier_applies_for_an_opportunity.feature [hunk 1]
```diff
@@ -253,7 +253,7 @@ Scenario: Previous page links are used during response flow and existing data is
   And I see '27/12/17' as the value of the 'availability' field
   And I don't see the 'Back to previous page' link
 
-
+@skip-preview
```
Comment (lfdebrux) [inline features/supplier/supplier_applies_for_an_opportunity.feature:256]: You might also want to skip this locally

```suggestion
@skip-local @skip-preview
```

File: features/supplier/supplier_applies_for_an_opportunity.feature [hunk 2]
```diff
@@ -335,7 +335,89 @@ Scenario: Supplier changes their answers before submission
...
+  Then I am on the 'What happens next' page
+  And I see a success flash message containing 'Your application has been submitted.'
+
+  When I click the 'View your account' link
+  And I click the 'View your opportunities' link
+  Then I see 'Tea drinker' in the 'Applications you’ve made' table
+  And I see the closing date of the brief in the 'Applications you’ve made' table
+  And I see 'Submitted' in the 'Applications you’ve made' table
+
+@skip-preview
```
Comment (lfdebrux) [inline features/supplier/supplier_applies_for_an_opportunity.feature:420]:
```suggestion
@skip-local @skip-preview
```

File: features/supplier/supplier_applies_for_an_opportunity.feature [hunk 3]
```diff
@@ -381,6 +463,52 @@ Scenario: Supplier changes their answers after submission
...
+  And I click the 'View your opportunities' link
+  Then I see 'Tea drinker' in the 'Applications you’ve made' table
+  And I see the closing date of the brief in the 'Applications you’ve made' table
+  And I see 'Submitted' in the 'Applications you’ve made' table
+  And I see 'You don’t have any draft applications' text on the page
+
+  When I click the 'Tea drinker' link
+  Then I am on the 'Your application for ‘Tea drinker’' page
+
+@skip-preview
```
Comment (lfdebrux) [inline features/supplier/supplier_applies_for_an_opportunity.feature:511]:
```suggestion
@skip-local @skip-preview
```

File: features/supplier/supplier_applies_for_an_opportunity.feature [hunk 4]
```diff
@@ -443,7 +601,7 @@ Scenario: Supplier can see the link to the opportunities dashboard
   When I click the 'View your account' link
   Then I see the 'View your opportunities' link
 
-@opportunities-dashboard
+@opportunities-dashboard @skip-preview
```
Comment (lfdebrux) [inline features/supplier/supplier_applies_for_an_opportunity.feature:604]:
```suggestion
@opportunities-dashboard
@skip-local @skip-preview
```

**Comment by lfdebrux** [comment]
Commit works for me locally

**Comment by alex9smith** [comment]
This might be a more general question, but what's the rationale behind skipping some of the tests on different environments?

**Comment by domoscargin** [comment]
> This might be a more general question, but what's the rationale behind skipping some of the tests on different environments?

For this PR, the old summary tables have separate captions - "Completed opportunities" and "Draft opportunities".

With the new tables, the caption for the table is now also the heading - "Applications you've made" and "Applications you've started".

So any environment with the old tables are expecting to find those captions, but they don't exist after https://github.com/alphagov/digitalmarketplace-brief-responses-frontend/pull/252

Could potentially rejig the test so it looks for the first `<h>` tag before a table if it doesn't find the right caption, but I think there's some room for error there.

**Review by lfdebrux — APPROVED** [review]

**Pull request merged**